OpenEdXMobile/res/values/strings.xml

```diff
@@ -570,6 +570,8 @@
     <string name="notification_toggle">Allow Notifications</string>
     <!-- Placeholder title for course content with no title -->
     <string name="untitled_block">Untitled</string>
+    <!-- Title for default channel notifications -->
+    <string name="default_notification_channel_title">edX Notification</string>
```

Comment:
Nits:

- `tools:ignore="MissingTranslation"` should be added to this string, since we don't have its Spanish translation yet.
- instead of `edX` we should always use the `{platform_name}` placeholder and escape it inside the code.